client/locations-list-map.js

```diff
@@ -1701,6 +1701,21 @@ $(() => {
 
     updateFilters(filters);
 
+    // Translate ?id= param to ?coords= param.
+    const { id } = searchParams;
+    if (id) {
+      const headers = result.values[1];
+      const rowIdIndex = headers.indexOf('row_id');
+      const latIndex = headers.indexOf('lat');
+      const lngIndex = headers.indexOf('lng');
+      const entry = result.values.find((element) => element[rowIdIndex] === id);
+      if (entry) {
+        searchParams.coords = `${entry[latIndex]},${entry[lngIndex]}`;
+      } else {
+        console.warn(`Found no row with id ${id}.`);
```

Comment:
@linderr @4kbt Do we have a "row_id" column for any country besides the US? If not can we add? It's been a while since I've added a column but I don't think it'd impact anything

Reply:
I don't think so - I can go through all non-US datasets and add tonight